samples/LowAllocationWebServer/LowAllocationWebServer.csproj

```diff
@@ -5,6 +5,14 @@
     <OutputType>Exe</OutputType>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Remove="LowAllocationWebServerCore\obj\**" />
+    <Compile Remove="LowAllocationWebServerLibrary\obj\**" />
+    <EmbeddedResource Remove="LowAllocationWebServerCore\obj\**" />
+    <EmbeddedResource Remove="LowAllocationWebServerLibrary\obj\**" />
+    <None Remove="LowAllocationWebServerCore\obj\**" />
+    <None Remove="LowAllocationWebServerLibrary\obj\**" />
+  </ItemGroup>
+  <ItemGroup>
```

Comment:
I don't understand this one either. It might be a byproduct of VS2017. However, anytime I built, somehow the obj files got pulled into the project and then caused failures. This was an easy fix with no real bad consequences.